### PR TITLE
Made  Navbar Responsive

### PR DESCRIPTION
**Pull request by HarshikaAdarsh**
## Description

Fixes: #16 

_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue._

Updated client/src/components/Navbar/index.tsx and client/src/component
s/Navbar/index.css to make navbar responsive.

## Type of change

_What type of changes does your code introduce to the wharf? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [x] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Verification

Please check the [testing guidelines](../docs/TESTING.md) for recommendations about automated tests.
I built the project locally and tested the changes. It works as expected.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,7 +8,7 @@ get:
 
 
 runFrontend: get
-	@npm --prefix ./pkg/client start
```
Comment (yp969803) [inline Makefile:11]: change it to previous value

**Comment by yp969803** [comment]
run 'make format'

**Comment by yp969803** [comment]
react build is having errors please solve it

**Comment by HarshikaAdarsh** [comment]
@yp969803 
I have fixed all the issues. Please review the changes.

**Comment by yp969803** [comment]
Thanks!

**Pull request merged**